eng/finalize-publish.yml

```diff
@@ -12,12 +12,27 @@ parameters:
   versionsRepo: versions
 
 jobs:
+  - job: waitForValidation
+    displayName: Wait for external validation
+    pool: server
+    timeoutInMinutes: 1440 # job times out in 3 days
+    steps:
+    - task: ManualValidation@0
+      timeoutInMinutes: 1440 # task times out in 1 day
+      inputs:
+        notifyUsers: |
+          juhoyosa@microsoft.com
```

Comment:
Maybe other diagnostics team members should be notified?

Reply:
> Maybe other diagnostics team members should be notified?

+1.  Please add me too :)

Reply:
I considered it, and it's fine as long as whoever does it does the verification that the one that's getting published actually matches the one being published.